src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -2,11 +2,7 @@
 
 public class BinaryString {
 
-    /**
-     * Напишите метод toBinaryString(), который принимает на вход целое число value,
-     * а возвращает String с представлением этого числа в двоичном виде.
-     */
     public String toBinaryString(int value) {
-        return null;
+        return Integer.toBinaryString(value);
```

Comment:
please, be careful:
[Create your own implementation, don’t use standard method](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_01.html)

Reply:
I have redone it.